js/pihole/queries.js

```diff
@@ -40,6 +41,33 @@ $(document).ready(function() {
           add(data[2],"black");
         }
     } );
+
+    // Do we want to filter queries?
+    var GETDict = {};
+    location.search.substr(1).split("&").forEach(function(item) {GETDict[item.split("=")[0]] = item.split("=")[1];});
+    if("client" in GETDict)
+    {
+        if(GETDict["client"] === "localhost(127.0.0.1)")
+        {
+            // Have to use normal search, as regexp of DataTable is broken
+            // It should be fixed in next release which might come up next
+            // year. In the meantime, we work around using normal search.
+            tableApi.column(3).search("localhost");
```

Comment:
This is only seen with "localhost(127.0.0.1)". Inspired from the now sanitize function for the domain names, I added something new and now this always works (no exceptions required anymore).